kong/pdk/log.lua

```diff
@@ -275,10 +275,11 @@ local function gen_log_func(lvl_const, imm_buf, to_string, stack_level, sep)
   to_string = to_string or tostring
   stack_level = stack_level or 2
 
-  local sys_log_level
   local variadic_buf = {}
 
   return function(...)
+    local sys_log_level = nil
+
```

Comment:
is adding the API to PDK at this scope?

Reply:
yes, dynamically changing to the log level should be reflected in the kong.log PDK